docs/development/decision-records/2024-08-28-Release-Workflow-Refactor/README.md

```diff
@@ -0,0 +1,82 @@
+# Refactor of the Release Workflow
+
+## Decision
+
+Refactor the existing workflows to accommodate the release of bugfix versions as described in [this](../2024-07-18-Improvements-versioning-scheme/README.md) decision record:
```

Comment:
minor nit: I would phrase this as "the existing workflows will be refactored to...."